src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -1148,6 +1148,9 @@ namespace OutputReportTabular {
                 curVariMeter = UtilityRoutines::MakeUPPERCase(MonthlyFieldSetInput(FirstColumn + colNum - 1).variMeter);
                 // call the key count function but only need count during this pass
                 GetVariableKeyCountandType(state, curVariMeter, KeyCount, TypeVar, AvgSumVar, StepTypeVar, UnitsVar);
+                if (TypeVar == OutputProcessor::VarType_NotFound) {
+                    ShowWarningError("In Output:Table:Monthly '" +  MonthlyInput(TabNum).name + "' invalid Variable or Meter Name '" + curVariMeter + "'");
+                }
```

Comment:
This is the actual warning. After going the manual route, I noticed that `GetVariableKeyCountandType` was already returning all the information needed to be able to throw a warning, so might as well use it here.